### PR TITLE
added dsadmin chart for administration of ds

### PR DESCRIPTION
**Pull request by wstrange**
script clean up

### CODE REVIEW

File: docker/ds/scripts/backup-admin.sh [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-# Revisit when https://bugster.forgerock.org/jira/browse/OPENDJ-4852 is fixed.
-
-
-source /opt/opendj/env.sh
-
-
-cd /opt/opendj/data
-
-# Create a unique folder for this host's backup.
-B="${BACKUP_DIRECTORY}/$HOSTNAME/admin"
```
Comment (wajihahmed) [inline docker/ds/scripts/backup-admin.sh]: Maybe call variable BAK instead of B?

File: docker/ds/scripts/schedule-backup.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+cd /opt/opendj
+
+source /opt/opendj/env.sh
+
+mkdir -p "$BACKUP_DIRECTORY"
+chmod 775 "$BACKUP_DIRECTORY"
+
+FULL_CRON="0 0 * * *"
+INCREMENTAL_CRON="0 * * *"
```
Comment (wajihahmed) [inline docker/ds/scripts/schedule-backup.sh]: Is there a "*" missing here?

Comment (wstrange) [reply docker/ds/scripts/schedule-backup.sh]: good catch

File: docker/ds/scripts/backup-admin.sh (hunk 1 above)
Comment (wstrange) [reply docker/ds/scripts/backup-admin.sh]: its deleted

File: docker/ds/scripts/export-ldif.sh [hunk 3]
```diff
@@ -2,19 +2,21 @@
...
 source /opt/opendj/env.sh
 
-# The parent path must exist on the remote host.
-BACKUP_DIRECTORY=${BACKUP_DIRECTORY:-/opt/opendj/bak}
-
 
 echo "Exporting LDIF"
 
-for root in "userRoot" "ctsRoot"; do
+# TOOD: The calculation of roots only works within a single instance.
```
Comment (shankar-forgerock) [inline docker/ds/scripts/export-ldif.sh]: I guess this is "TODO"

Comment (dgoldssfo) [inline docker/ds/scripts/export-ldif.sh]: DOOD, I think you mean TODO

Comment (shankar-forgerock) [reply docker/ds/scripts/export-ldif.sh]: DG: "DOOD" Did you mean Dude or TOOD?

**Pull request merged**